src/PlaywrightSharp/StorageState.cs

```diff
@@ -5,7 +5,7 @@
 namespace PlaywrightSharp
 {
     /// <summary>
-    /// See <see cref="BrowserContextOptions.StorageState"/>.
+    /// TODO: move this.
```

Comment:
Is this still a todo?

Reply:
Yes, this will disappear when we introduce the new `BrowserContext`.